### PR TITLE
added swagger file

### PR DESCRIPTION
**Pull request by radgupte**
### Reviewer: Sumer Patel
Added a swagger doc file to a separate branch and initiated a pull request to merge with the main branch

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -0,0 +1,358 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is a Weather server",
+    "version": "1.0.0",
+    "title": "Weather"
+  },
+  "basePath": "/v2",
```
Comment (Sumer16) [inline swagger.json:8]: use /v1 instead of /v2

File: swagger.json [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+      "name": "v1",
+      "description": "Weather App"
+    }
+  ],
+  "schemes": [
+    "https",
+    "http"
+  ],
+  "paths": {
+    "/v1/weather": {
```
Comment (Sumer16) [inline swagger.json]: remove /v1

File: swagger.json (hunk 1 above)
Comment (Sumer16) [inline swagger.json]: change it to /v1

File: swagger.json [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+          {
+            "petstore_auth": [
+              "write:pets",
+              "read:pets"
+            ]
+          }
+        ]
+      }
+    },
+    "/v1/hello": {
```
Comment (Sumer16) [inline swagger.json]: remove /v1

File: swagger.json [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+          {
+            "petstore_auth": [
+              "write:pets",
+              "read:pets"
+            ]
+          }
+        ]
+      }
+    },
+    "/v1/auth": {
```
Comment (Sumer16) [inline swagger.json]: remove /v1

File: swagger.json [hunk 5]
```diff
@@ -0,0 +1,358 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is a Weather server",
```
Comment (Sumer16) [inline swagger.json:4]: Add a good description

File: swagger.json [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+  },
+  "basePath": "/v2",
+  "tags": [
+    {
+      "name": "v1",
+      "description": "Weather App"
+    }
+  ],
+  "schemes": [
+    "https",
```
Comment (Sumer16) [inline swagger.json]: remove "https" from here

**Review by Sumer16 — CHANGES_REQUESTED** [review]
Please make these changes for the swagger.json file

File: swagger.json (hunk 2 above)
Comment (radgupte) [reply swagger.json]: Removed /v1

File: swagger.json (hunk 1 above)
Comment (radgupte) [reply swagger.json]: changed

File: swagger.json (hunk 3 above)
Comment (radgupte) [reply swagger.json]: removed

File: swagger.json (hunk 4 above)
Comment (radgupte) [reply swagger.json]: removed

File: swagger.json (hunk 5 above)
Comment (radgupte) [reply swagger.json:4]: added

File: swagger.json (hunk 6 above)
Comment (radgupte) [reply swagger.json]: removed

**Review by Sumer16 — APPROVED** [review]
All the changes are good, congratulations on your first PR!

**Pull request merged**